src/test/config.sol

```diff
@@ -94,9 +94,9 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:         address(0x622Ad624491a01a2a6beAD916C3Ca3B90BcA0854), // populate with deployed spell if deployed
-            deployed_spell_created: 1718197007,          // use `make deploy-info tx=<deployment-tx>` to obtain the timestamp
-            deployed_spell_block:   20075890,          // use `make deploy-info tx=<deployment-tx>` to obtain the block number
+            deployed_spell:         0x7fbC867dE58D6e47E430eB257B50481F6E878f65, // populate with deployed spell if deployed
```

Comment:
Please use `address()` here, as per [crafters checklist](https://github.com/makerdao/pe-checklists/blob/81ed24f37b477a07bb59abd8cd60f6049ad4fc1c/spell/spell-crafter-mainnet-workflow.md?plain=1#L184)

Reply:
done in [6903093](https://github.com/makerdao/spells-mainnet/pull/414/commits/69030932e0662be97d5c76028e481955ab04e00a)